### DISCUSSION TITLE
[Show and Tell] Mint zkSync NFTs with customizable dapp for your website and brand

### DISCUSSION THREAD

**Discussion by 0xiliyan** [zkSync-Community-Hub/zksync-developers#383] [Show and Tell]
### Project Name

zkSync No-Code Minting Dapp

### Repo Link

https://github.com/0xiliyan/zksync-minting-dapp

### Demo Link (Optional)

Tutorial will follow soon 📌

### Technologies Used

Next.js, React, ThirdWeb SDK

### Describe your project. What does it do? Why did you create it? Any interesting challenges you faced?

📢 **Create a branded minting dapp for your zkSync project without any coding skills** 📢 

If you have idea for a project - introduce NFT gated content, make a PFP collection, whatever it might be ✅
And you want to create a dapp that blends with your existing website or branding content ✅
But lack any coding skills or only know a bit of HTML / CSS ❌

Then this repository will help you without having to rely on external platforms that are hosted on third party domains or restrict the customization level of your dapp 🚀

### What specific feedback are you looking for?

I'm looking to find out what people needs are exactly, lots of people are looking for a highly customizable minting dapp to integrate with their project and host it themselves, not on external third-party platform

### How can the community help or contribute to your project?

Report bugs or suggest new feature implementations

**Comment by eminagov** [comment]
Looks pretty simple to config and I am sure it will be useful for speeding up development.

**Reply by Malachy999** [reply]
any further developments on this?

**Reply by 0xiliyan** [reply]
Yes I plan to add more configuration options to the panel in time

**Comment by sundaramjacob** [comment]
Introducing zkSync No-Code Minting Dapp!

🚀 Create a custom minting dapp for your zkSync project effortlessly.
💡 Perfect for projects needing seamless integration and self-hosted solutions.

🔍 Seeking feedback on usability and feature suggestions.
💬 Community contributions welcome for bug reports and new features.

Let's simplify NFT minting with zkSync No-Code Minting Dapp!

**Comment by jennifercerso** [comment]
The zkSync No-Code Minting Dapp is a project that allows users to create a branded minting dapp for their zkSync project without any coding skills. The project uses Next.js, React, and ThirdWeb SDK. The aim of the project is to provide a highly customizable minting dapp that can be integrated with a user's project and hosted on their own platform, rather than relying on external third-party platforms. The creator is looking for feedback on user needs and suggestions for new features, as well as bug reports.

**Comment by andrewtotta** [comment]
The zkSync No-Code Minting Dapp is a Next.js and React-based project that allows users to create a branded minting dapp for their zkSync project without any coding skills. The dapp can be customized to blend with existing websites or branding content, and users can integrate it with their project and host it themselves. The creator is looking for feedback on user needs and suggestions for new feature implementations, and the community can help by reporting bugs and contributing to the project.
